File: archive-common/src/main/java/site/archive/common/exception/user/OAuthRegisterFailException.java
package site.archive.common.exception.user;

import site.archive.common.exception.BaseException;
import site.archive.common.exception.ExceptionCode;

public class OAuthRegisterFailException extends BaseException {

    public OAuthRegisterFailException(String oAuthProviderRegistrationId, String message) {
        super(String.format("%s : %s", oAuthProviderRegistrationId, message),
              ExceptionCode.REGISTER_FAIL);
    }

}

File: archive-common/src/main/kotlin/site/archive/common/exception/user/OAuthRegisterFailException.kt
package site.archive.common.exception.user

import site.archive.common.exception.BaseException
import site.archive.common.exception.ExceptionCode

class OAuthRegisterFailException(oAuthProviderRegistrationId: String, message: String) :
    BaseException("$oAuthProviderRegistrationId : $message", ExceptionCode.REGISTER_FAIL)
